west.yml

```diff
@@ -257,7 +257,7 @@ manifest:
       path: modules/lib/zcbor
     - name: zscilib
       path: modules/lib/zscilib
-      revision: ca070ddabdaf67175a2da901d0bd62e8899371c5
+      revision: pull/45/head
```

Comment:
This needs to be changed to the commit hash now that the other PR has been merged.

Reply:
It is already changed